src/SuperDumpService/Services/DumpRepository.cs

```diff
@@ -119,8 +119,10 @@ public class DumpRepository {
 		internal void SetDumpStatus(string bundleId, string dumpId, DumpStatus status, string errorMessage = null) {
 			var dumpInfo = Get(bundleId, dumpId);
 			dumpInfo.Status = status;
-			if (!string.IsNullOrEmpty(errorMessage)) {
-				dumpInfo.ErrorMessage = errorMessage;
+			dumpInfo.ErrorMessage = errorMessage;
```

Comment:
If an analysis fails, the error message is set accordingly. However, if a reanalyze succeeds, the error message doesn't get reset with the old implementation. This ensures that the error message is up to date.